packages/vscode-extension/.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+	// See http://go.microsoft.com/fwlink/?LinkId=827846
```

Comment:
I don't think we should have comments in .json files, this syntax is not valid

Reply:
It was automatically added int the scafolding.